spec/array_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "./lib/array.rb"
+
+
+describe Array do
+	it "add all the values in array [1,2,3,4]" do
+		expect([1,2,3,4].rekt { |sum, number| sum + number}).to eq(10)
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Space missing after comma.<br>Space missing inside }.<br>Tab detected.